tools/config-generator/main.go

```diff
@@ -205,6 +204,11 @@ func newbaseProwJobTemplateData(repo string) baseProwJobTemplateData {
 	data.Timeout = 50
 	data.OrgName = strings.Split(repo, "/")[0]
 	data.RepoName = strings.Replace(repo, data.OrgName+"/", "", 1)
+	data.ExtraRefs = []string{"- org: " + data.OrgName, "  repo: " + data.RepoName}
+	if data.OrgName == "knative" || data.OrgName == "knative-sandbox" {
+		data.PathAlias = "path_alias: knative.dev/" + data.RepoName
```

Comment:
Not quite sure, but I'm afraid that not every repo is on knative.dev yet. At least docs is not yet
https://github.com/knative/website/blob/master/static/_redirects

Reply:
Thanks for noticing this. It looks docs is the only repo that is still in `dep` and not using path alias.. 
I have added some extra logic in the config generator tool, which makes the code dirty but also makes the meta config file much cleaner.

Reply:
How about move docs over to knative.dev? It's not too much code

Reply:
That will break the Prow jobs for the release branches, as they are not in Go modules and have to be under `GOPATH`. To be backward compatible, we'll have to get back the per branch config generation logic that I deleted in https://github.com/knative/test-infra/pull/2148, which I don't want to..

I'm trying to switch docs to Go mod - https://github.com/knative/docs/pull/2718, if it can be merged, we will be able to delete this special case in the config generator after 4 release cycles.

Reply:
ack